app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -0,0 +1,422 @@
+// non-jquery version of magna-charta: https://github.com/alphagov/magna-charta
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function MagnaCharta () { }
+
+  MagnaCharta.prototype.start = function ($module, options) {
+    this.$module = $module[0]
+    this.options = {
+      outOf: 65,
+      applyOnInit: true,
+      toggleText: 'Toggle between chart and table',
+      autoOutdent: false,
+      outdentAll: false,
+      toggleAfter: false, // BOOL set TRUE to append the toggle link
+      returnReference: false // for testing purposes
+    }
+
+    for (var k in options) this.options[k] = options[k]
+    this.detectIEVersion()
+
+    // if it's IE7 or less, we just show the plain tables
+    this.ENABLED = !(this.ie && this.ie < 8)
```

Comment:
With the use of `classList` meaning that only IE 11 + is able to run this JavaScript, should this be changed to only be `ENABLED` when in IE 11 + and not IE? 

(This is complicated by IE10 not supporting conditional comments when in standards mode, so the `detectIEVersion()` function probably won't work.)

Reply:
Strangely, this does all work in IE8+, however I think it's a weird coincidence. The step by step component JS now imports a polyfill for classlist, and I think it's being initialised before this (because I've tested it in IE8 and it's working). So I've explicitly added in the polyfill to this file as well.

Reply:
Super!